pkg/service/service.go

```diff
@@ -933,6 +933,33 @@ func (d *DynamicCRD) AddComponentLabelsToCRD(labels map[string]string) {
 	metaMap["labels"] = labels
 }
 
+func (d *DynamicCRD) GetGVK() (group, version, kind string, err error) {
+	apiVersion, ok := d.OriginalCRD["apiVersion"].(string)
+	if !ok {
+		err = errors.New("couldn't parse GVK information")
+		return
+	}
+
+	apiVersionList := strings.SplitN(apiVersion, "/", 2)
```

Comment:
you should use methods from apimachinery to parse groupvesrions instead of writing your own https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime/schema#ParseGroupVersion